pretix_sepadebit/models.py

```diff
@@ -15,3 +16,11 @@ class SepaExportOrder(models.Model):
     order = models.ForeignKey('pretixbase.Order', on_delete=models.CASCADE)
     payment = models.ForeignKey('pretixbase.OrderPayment', on_delete=models.CASCADE, null=True)
     amount = models.DecimalField(max_digits=10, decimal_places=2)
+
+
+class SepaDueDate(models.Model):
+    payment = models.OneToOneField('pretixbase.OrderPayment', on_delete=models.CASCADE, null=True, related_name='due')
+    date = models.DateField()
+    remind_after = models.DateTimeField()
+    reminded = models.BooleanField(verbose_name=_("Reminder"), default=False)
```

Comment:
The verbose name isn't really clear, "pre-notification sent" would be clearer. However, since this isn't ever shown to a user, we probably don't need it at all?